app.R

```diff
@@ -931,18 +916,120 @@ ui <- fluidPage(
     ),
     
     # upload data ----
-    tabPanel(
-      title = div("Create Your Own MWI", class = "explore"),
-      fluidRow(
-        column(width = 2),
-        column(
-          width = 8,
-          HTML("<center><h2>Create your own Mental Wellness Index (MWI)!</h2></center>"),
-          HTML(paste0(
-            "<p align = 'justify'>",
-            "To create your own Mental Wellness Index, you must be running the Mental Wellness Index Tool on your local computer in order to protect your data. Follow the instructions below to create your own MWI for your community below by adjusting weights and/or adding your own data and metadata.",
-            "<ol>",
-            if (!app_local){
+    
+    navbarMenu(
+      "Create Your Own MWI",
+      tabPanel(
+        title = div("Adjust MWI Weights", class = "explore"),
+        fluidRow(
+          column(width = 2),
+          column(
+            width = 8,
+            HTML("<center><h2>Change weights to create your own Mental Wellness Index (MWI)!</h2></center>"),
+            HTML(paste0(
+              "<p align = 'justify'>",
+              "Weights used in the Mental Wellness Index control the relative influence each measure has on the total MWI; higher numbers inidcate a higher influence. If you adjust the weights to 0, a measure has no influence on the MWI.",
+              "<br><br>",
+              "To adjust the weights in the Mental Wellness Index, follow the instructions below. If you want to add your own data to the MWI, go to the \"Add Local Data to MWI\" section. Note that data uploaded to this application is not kept -- it is deleted once you leave the page, including any processing done to it.",
+              "<ol>",
+              "<li>Update weights for each measure in the table below as desired by doubleclicking the 'Updated Weights' column, then editing the measure to the desired amount (0 or a positive number).</li>",
```

Comment:
Maybe another note that the weights do not need to add to 100 (optional)

Reply:
done!